testing/tests/filecheck.test.ts

```diff
@@ -1,23 +1,23 @@
 import fs from "node:fs";
-import path from "node:path";
 
 import { checkFile } from "../../filecheck/checker";
 
-const dirname = __dirname;
-
-const SAMPLES_DIRECTORY = path.join(dirname, "filechecker", "samplefiles-html");
+const SAMPLES_DIRECTORY = new URL(
+  "filechecker/samplefiles-html/",
+  import.meta.url
+);
 
 describe("checking files", () => {
   it("should spot SVGs with scripts inside them", async () => {
-    const filePath = path.join(SAMPLES_DIRECTORY, "script.svg");
+    const filePath = new URL("./script.svg", SAMPLES_DIRECTORY);
```

Comment:
@queengooborg This change causes the ["Testing Yari" workflow to fail](https://github.com/mdn/yari/actions/runs/3590827297/jobs/6044648259#step:7:24), because `filePath` was a `string` and is now a `URL`. We need to convert it back to a `string`.

Besides, I feel like this will cause issues on Windows. 🤔

Reply:
Wrapping it with `fileURLToPath()` should fix it I guess.